ingest/ingest.go

```diff
@@ -410,6 +416,7 @@ func (i *FileIngest) Stop() error {
 	fmt.Println("Running cleanup tasks...")
 
 	// TODO: set readtimeout to something besides 0 to close keepalive connections
+	i.app.Server().ReadTimeout = 30 * time.Second
```

Comment:
Let's think about making this change in a separate PR. The value should be configurable as well.